pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
```

Comment:
Regarding the imports sorting, I'd add either:

```toml
[tool.ruff.lint.isort]
known-first-party = ["apify"]
```

or even include all our pckgs:
```toml
[tool.ruff.lint.isort]
known-first-party = ["apify", "apify_client", "apify_shared", "crawlee]
```

to distinguish from 3rd party pckgs.